docs/decisions/0001-phase-in-db-backed-organizations-to-all.rst

```diff
@@ -181,11 +181,11 @@ These changes will be made in `pull request 25153`_.
 Community
 ~~~~~~~~~
 
-The community will be informed of the change via both Discourse and the release notes for Koa.
+The community will be informed of the change via both Discourse and the release notes for Lilac.
 Instructions for migration will be included.
 
 `DEPR-117`_ is created to track the eventual removal of the ``FEATURES["ORGANIATIONS_APP"]``,
-ideally in Lilac.
+ideally in Maple.
```

Comment:
```suggestion
which will also be part of Lilac.
```

Whoops, I had changed my mind about this and never updated the ADR.

I decided against supporting ``FEATURES["ORGANIATIONS_APP"]`` alongside the new toggle, as it introduced edge cases when both toggles were set. So, it'll also be gone in Lilac.

Reply:
Ha! So much for me searching out the M* release name 😛